.travis.yml

```diff
@@ -1,5 +1,3 @@
-sudo: false
```

Comment:
This isn't needed anymore since the end of last year as all Travis builds now run on virtual machines. See https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration